iotedgehubdev/cli.py

```diff
@@ -269,10 +269,22 @@ def stop(host):
     output.info('IoT Edge Simulator has been stopped successfully.')
 
 
+@click.command(context_settings=CONTEXT_SETTINGS,
+               help="Determine whether config file is valid.")
+@_with_telemetry
+def validateconfig():
+    try:
```

Comment:
no need to try/catch here. _parse_config_json already handle this right?

Reply:
yes, but it contains other information, such as 'please run setup first'

Reply:
Updated